### ISSUE TITLE
"Error: Could not open dictionary "demo-sql/4.0.dict" when building from latest git source

### ISSUE THREAD

**Issue by alexei-gl** [opencog/link-grammar#794]
I get "Error: Could not open dictionary "demo-sql/4.0.dict" when building from latest git source code. Log  files are attached.
[build-issue.tar.gz](https://github.com/opencog/link-grammar/files/2169181/build-issue.tar.gz)

**Comment by ampli** [comment]
It seems you need to install the SQLite development package.
From your build.log:
```
checking sqlite3.h usability... no
checking sqlite3.h presence... no
checking for sqlite3.h... no
```

And then in the configuration setting summary:
```
        SQLite-backed dictionary:       no
```

**Comment by ampli** [comment]
I just sent a PR (#797) to install it.

BTW, for some reason GitHub doesn't give an option to get notifications for comment updates, so it may be a good idea to post a new comment for non-typo updates.

**Comment by linas** [comment]
I merged #797 please test and close if that resolves the issue.

**Comment by alexei-gl** [comment]
I just tested. The issue is resolved. Thank you!
